src/model/mosaic/MosaicId.ts

```diff
@@ -43,20 +63,6 @@ export class MosaicId {
         if (id instanceof Array) {
```

Comment:
is the id: string still required?

Reply:
As discussed in private - `string` capacity allows for hexadecimal notation initialization :+1: